front/app/translations/admin/en.json

```diff
@@ -283,6 +283,7 @@
   "app.containers.Admin.Insights.Reports.inputs": "Posts",
   "app.containers.Admin.Insights.Reports.inputsByStatusTitle": "Posts by status",
   "app.containers.Admin.Insights.Reports.month": "month",
+  "app.containers.Admin.Insights.Reports.native_survey": "native survey",
```

Comment:
Do these changes belong here @sebastienhoorens?

Reply:
I didn't make these changes, this happened through an automatic commit, so I don't know 😅

Reply:
Merging in latest master solved it